CONTRIBUTING.md

```diff
@@ -47,15 +47,33 @@ This command will list all the suites and options for running tests.
 yarn test
 ```
 
-_Note that in order to run the tests fro ReactNative, you must have bootstrapped with ReactNative enabled_
+The options for running tests can be selected from the cli or be passed to `yarn test` with specific parameters.  Available modes include `--watch`, `--coverage`, and `--runInBand`, which will respectively run tests in watch mode, output code coverage, and run selected test suites serially in the current process.
```

Comment:
Please mention the `--update` option here as well. It works for all the tests, not only e2e